src/app/components/LiteSiteCta/index.tsx

```diff
@@ -0,0 +1,100 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import { RequestContext } from '../../contexts/RequestContext';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import styles from './index.styles';
+import { defaultTranslations } from './liteSiteConfig';
+
+type CtaLinkProps = {
+  isRtl: boolean;
+  href: string;
+  text: string;
+  fontVariant?: string;
+  showChevron?: boolean;
+  ignoreLiteExtension?: boolean;
+  className?: string;
+};
+
+const CtaLink = ({
+  isRtl,
+  href,
+  text,
+  fontVariant = 'sansRegular',
+  showChevron = false,
+  ignoreLiteExtension = false,
+  className,
+}: CtaLinkProps) => {
+  const chevron = isRtl ? (
+    <LeftChevron css={styles.chevron} />
+  ) : (
+    <RightChevron css={styles.chevron} />
+  );
+
+  return (
+    <a
+      href={href}
+      className={className}
+      css={styles.link}
+      {...(ignoreLiteExtension && { 'data-ignore-lite': true })}
+    >
+      <Text size="brevier" fontVariant={fontVariant} css={styles.linkText}>
+        {text}
+      </Text>
+      {showChevron && chevron}
+    </a>
+  );
+};
+
+const LiteSiteCta = () => {
+  const { dir, translations } = useContext(ServiceContext);
+  const { canonicalLink } = useContext(RequestContext);
+  const isRtl = dir === 'rtl';
+  const { liteSite = defaultTranslations } = translations;
+  const { onboardingMessage, toMainSite, informationPage, dataSaving } =
+    liteSite;
+  const id = 'LiteSiteCta';
+
+  return (
+    <section
+      role="region"
+      data-e2e="lite-cta"
+      aria-labelledby={id}
+      css={styles.outerContainer}
+    >
+      <VisuallyHiddenText as="strong" id={id} aria-hidden>
```

Comment:
Could we use the 'hidden' attribute rather than aria hidden and visualy hidden CSS please? https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden (aria-hidden attribute hides content from assistive technology but doesn't visually hide anything hence your use of the visualy hidden CSS in addition, the hidden attribute does both these things)

Reply:
Can do those this also seems to hide the strong element in no-css mode (using the Web Developer tool bookmark). Is this ok?

As it is
![Screenshot 2024-10-23 at 12 28 15](https://github.com/user-attachments/assets/64db4acd-8e36-48cc-a8e6-a5c8f830b747)


If I change to <Text... hidden>
![Screenshot 2024-10-23 at 12 27 56](https://github.com/user-attachments/assets/b247f639-9bfc-4433-a62b-cf030f768e8a)

Reply:
Interesting, thanks. I'm not sure if the text content will be available to the aria-labelledby if it's aria-hidden?

Reply:
I've tested it on Safari and region is announced labelled by the text in the strong area. Admittedly I won't know about all other AT until the swarm.

I'm happy to change it to `hidden` if you think that's better. I'll just add a note by this A11y AAC since we won't meet this criteria:  https://paper.dropbox.com/doc/Lite-Canonical-site-CTA-Accessibility-acceptance-criteria--CZMCHU_W8cst0_HvvG47gEVSAg-xyTULgByamiYbQSOrzd9h#:h2=No-CSS

Reply:
As long as this criteria is met I don’t mind e.g. we don’t want ‘Data saving’ announced as it’s own element when you navigate through the component. Please update the AAC accordingly.

- Given I am using a screen reader
- And I am viewing the lite site
- And I navigate to the lite/canonical site CTA component
- Then the region landmark is announced
- And the region’s label is announced once
- Then the next element that is announced is the paragraph of text

Reply:
Fab have updated